test/saxon-custom-options/test.xspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<x:description query="x-urn:test:do-nothing" query-at="../do-nothing.xquery"
+	stylesheet="../do-nothing.xsl" xmlns:do-nothing="x-urn:test:do-nothing"
+	xmlns:x="http://www.jenitennison.com/xslt/xspec">
+	<x:scenario label="Compare 'a' and 'A' using custom collation">
+		<x:call function="compare">
+			<x:param select="'a'" />
+			<x:param select="'A'" />
+			<x:param select="'x-urn:test:saxon-custon-options:collation'" />
```

Comment:
I think there is a typo here, it should be:

```
uri="x-urn:test:saxon-custom-options:collation"
```

Note `custom` instead of `custon` (which I believe is a typo).